.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
   },
   globals: {
     d3: true,
+    Tour: true
```

Comment:
Prefer create an ESLint configuration file in `packages/manager/apps/public-cloud/` and extend from the root configuration.

For instance `script` part has already extending the root configuration (cf: https://github.com/ovh-ux/manager/blob/master/scripts/.eslintrc.json)